streams/ro.m3u

```diff
@@ -132,6 +134,8 @@ https://v1.mediastreaming.ro/stream/hls/ploiestitv.m3u8
 https://stream1.1616.ro:1945/prima/livestream/playlist.m3u8
 #EXTINF:-1 tvg-id="PROTVNews.ro",Pro TV News (1080p)
 https://cmero-ott-live.ssl.cdn.cra.cz/channels/cme-ro-voyo-news/playlist.m3u8?offsetSeconds=0&url=0
+#EXTINF:-1 tvg-id="ProTVInt.ro",Pro TV International (576p)
+http://romania4k.ddns.net:8008/loredana_nicolescu/cF5a5bDq3b/33794
```

Comment:
what do you think of that stream? @BellezaEmporium

Reply:
![image](https://github.com/iptv-org/iptv/assets/64033272/519a12f3-e9d6-4789-94e9-3fc4ec1a3430)

Reply:
Xtream-Codes.
![image](https://github.com/iptv-org/iptv/assets/30985701/eeede3ca-4894-47b3-a062-5121f25baca4)

Not made for more than 3 simultaneous persons. Won't be stable.